src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/queue/TempConfig.java

```diff
@@ -0,0 +1,46 @@
+package ca.bc.gov.open.jag.efilingreviewerapi.queue;
+
+import ca.bc.gov.open.clamav.starter.ClamAvService;
+import ca.bc.gov.open.efilingdiligenclient.diligen.DiligenService;
+import ca.bc.gov.open.jag.efilingreviewerapi.document.DocumentsApiDelegateImpl;
+import ca.bc.gov.open.jag.efilingreviewerapi.extract.mappers.ExtractRequestMapper;
+import ca.bc.gov.open.jag.efilingreviewerapi.extract.store.ExtractStore;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.springframework.data.redis.core.StringRedisTemplate;
+import org.springframework.data.redis.listener.PatternTopic;
+import org.springframework.data.redis.listener.RedisMessageListenerContainer;
+import org.springframework.data.redis.listener.adapter.MessageListenerAdapter;
+
+@Configuration
+@EnableCaching
+public class TempConfig {
+
+    @Bean
+    RedisMessageListenerContainer container(JedisConnectionFactory jedisConnectionFactory,
+                                            MessageListenerAdapter listenerAdapter) {
+
+        RedisMessageListenerContainer container = new RedisMessageListenerContainer();
+        container.setConnectionFactory(jedisConnectionFactory);
+        container.addMessageListener(listenerAdapter, new PatternTopic("documentWait"));
```

Comment:
Should be keys. But Since this code will be deleted soon I wanted minimize the cleanup.

Reply:
all good